### DISCUSSION TITLE
Disable dialog to confirm closing of terminal.

### DISCUSSION THREAD

**Discussion by aadi58002** [ghostty-org/ghostty#3660] [Q&A] [answered]
A config option to disable the confirmation dialog when you try to close the terminal.
![image](https://github.com/user-attachments/assets/3adfe4df-d0d2-4b1b-91e7-9fc7db275df9)

# Platform
Void linux
## Ghostty version info
Ghostty 1.0.0

Version
  - version: 1.0.0
  - channel: stable
Build Config
  - Zig version: 0.13.0
  - build mode : builtin.OptimizeMode.ReleaseFast
  - app runtime: apprt.Runtime.gtk
  - font engine: font.main.Backend.fontconfig_freetype
  - renderer   : renderer.OpenGL
  - libxev     : main.Backend.io_uring
  - GTK version:
    build      : 4.16.7
    runtime    : 4.16.7
  - libadwaita : enabled
    build      : 1.6.2
    runtime    : 1.6.2

**Comment by 00-kat** [comment, marked as answer]
https://ghostty.org/docs/config/reference#confirm-close-surface

**Reply by aadi58002** [reply]
Thanks
